File: examples/node/list.ts
import { SailupClient } from '@kalourmade/sailup-sms';

const client = new SailupClient(process.env.SAILUP_API_KEY!);

const page = await client.sms.list({ page: 1, pageSize: 10 });

for (const message of page.results) {
  console.log(`${message.id} — ${message.body} (${message.deliveryStatus})`);
}

File: examples/python/list.py
import os

from kalourmade_sailup import SailupClient

client = SailupClient(os.environ["SAILUP_API_KEY"])

page = client.sms.list(page=1, page_size=10)

for message in page.results:
    print(f"{message.id} — {message.body} ({message.delivery_status.value})")
